conf/evolutions/091-oidc.sql

```diff
@@ -0,0 +1,35 @@
+BEGIN transaction;
+
+DROP VIEW webknossos.userInfos;
+DROP VIEW webknossos.multiUsers_;
+
+-- Cannot alter enum in transaction block, workaround required
```

Comment:
Cool that this works :) When this happened previously I added the enum type outside of the transaction, which I think is also fine (since even if the rest of the transaction fails, this should not cause actual problems). But it is good to have this version, also as reference for the future.